images/build/Dockerfiles/Dockerfile

```diff
@@ -103,6 +103,7 @@ RUN set -ex \
     && $imagesDir/installPlatform.sh php $PHP70_VERSION \
     && $imagesDir/installPlatform.sh php $PHP72_VERSION \
     && $imagesDir/installPlatform.sh php $PHP74_VERSION \
+    && $imagesDir/installPlatform.sh php $PHP80_VERSION \
```

Comment:
we need to ask app service team if we  need to bake php 8 in the image or we should make it available only for dynamic install? ... also php_8 is buster based i believe .. so  even if we keep it in build image we may have problem running it in buster runtime

Reply:
As our discussion today, we'll make php 8 dynamic install and add it to the buster based build image.